### ISSUE TITLE
ReactOnRails::Utils.bundle_js_file_path returns incorrect path for manifest.json

### ISSUE THREAD

**Issue by elstgav** [shakacode/react_on_rails#1011]
I noticed that `ReactOnRails::TestHelper.ensure_assets_compiled` always reports that `manifest.json` is stale when running tests, even when the file exists and is up to date:

```
Detected the following stale generated files:
  manifest.json
```

After investigating, I believe this is because `ReactOnRails::Utils.bundle_js_file_path()` does not return the correct file path for `'manifest.json'`, so `ReactOnRails::TestHelper.ensure_assets_compiled` can’t accurately check if the file exists and is up to date.

Thanks to [line 89 of `lib/react_on_rails/utils.rb`](https://github.com/shakacode/react_on_rails/blob/master/lib/react_on_rails/utils.rb#L89), it will always return `'manifest.json'` instead of the correct path (e.g. `'/path/to/rails/public/packs-test/manifest.json'`):
```rb
def self.bundle_js_file_path(bundle_name)
  if using_webpacker?
    return bundle_name if bundle_name == "manifest.json"
```

So in summary:

### Problem:

`ReactOnRails::Utils.bundle_js_file_path('manifest.json')` 
returns `'manifest.json'`

### Expected:

`ReactOnRails::Utils.bundle_js_file_path('manifest.json')` 
returns `'/path/to/rails/public/packs-test/manifest.json'`

### Proposed Solution:

Change `ReactOnRails::Utils.bundle_js_file_path` to

```rb
def self.bundle_js_file_path(bundle_name)
  if using_webpacker? && bundle_name != "manifest.json"
    bundle_js_file_path_from_webpacker(bundle_name)
  else
    # Default to the non-hashed name in the specified output directory, which, for legacy
    # React on Rails, this is the output directory picked up by the asset pipeline.
    File.join(ReactOnRails.configuration.generated_assets_dir, bundle_name)
  end
end
```

**Comment by justin808** [comment]
@elstgav good work. Please submit a PR.
